packages/common/src/types.ts

```diff
@@ -108,26 +108,138 @@ export type IBatch = any[];
 export type IBatchTransform = (value: any) => any;
 
 export interface IGetSurveyModelsResponse {
-  form: IModel,
-  featureService: IModel,
-  fieldworker: IModel,
-  stakeholder: IModel
-};
+  form: IModel;
+  featureService: IModel;
+  fieldworker: IModel;
+  stakeholder: IModel;
+}
 
 export interface IGetGroupSharingDetailsResults {
-  group: IGroup,
-  modelsToShare: IModel[]
-};
+  group: IGroup;
+  modelsToShare: IModel[];
+}
 
 export interface IRevertableTaskSuccess {
-  status: "fullfilled",
-  revert: (...args: any[]) => Promise<any>,
-  results: any
-};
+  status: "fullfilled";
+  revert: (...args: any[]) => Promise<any>;
+  results: any;
+}
 
 export interface IRevertableTaskFailed {
-  status: "rejected",
-  error: Error
-};
+  status: "rejected";
+  error: Error;
+}
+
+export type IRevertableTaskResult =
+  | IRevertableTaskSuccess
+  | IRevertableTaskFailed;
+
+/**
+ * Hub Types
+ */
+export type HubType =
+  | "member"
+  | "team"
+  | "event"
+  | "dataset"
+  | "document"
+  | "map "
+  | "app "
+  | "site"
+  | "initiative"
+  | "template"
+  | "organization";
+export type VisibilityOptions = "private" | "org" | "public";
+export type ControlOptions = "view" | "edit" | "admin";
+
+// TW: basing this on https://opendata.arcgis.com/api/v3/datasets/7a153563b0c74f7eb2b3eae8a66f2fbb_0/
+// but that may not be accurate
+export interface IHubGeography {
+  size?: number; // what is this?
```

Comment:
I don't recall the details of why this was included (Dmfenton, pranav, alex, other interns). I don't see any notes in emails. I think we can drop it, unless you see a reference in live code.

This is similar to your dropping other speculative geography attributes (name, source) until they're explicitly added.

Reply:
thanks @ajturner. @bcamper do you know what the `boundary.size` property is?

Reply:
I think it is the actual size of the ring data?? Not sure why it is being stored and don't think it has a functional purpose.

But also, I believe the data structure for the concave hulls is changing (I had filed a bug about how it's neither compliant Esri JSON geometry nor GeoJSON) - @mikeringrose, do you know that status?

Reply:
I've removed `size` for now

Reply:
@mikeringrose @bcamper  we need to at least figure out what format the Hub API _intends_ to return the `boundary` before we merge this PR.

Reply:
TP issue I filed is here: https://esriarlington.tpondemand.com/entity/138758-incorrectly-formatted-concave-hull-geometries
PR in Hub API: https://github.com/ArcGIS/hub-indexer/pull/2242

I ran into this a few months ago when reusing some of the graphics sorting code for map notes, but abandoned that change because of this issue.

But I did want to ensure we're storing compliant geometries (and there were other oddities in the current format) before we expand the use of additional geometries with things like boundary picker UI etc.

Reply:
Thanks @bcamper. So it looks like the intent at least is to store/return Esri geometries, so I'll consider things resolved on this end - they should "just work" once the bug is fixed on the server.